tools/gen-jit-template.py

```diff
@@ -216,6 +216,7 @@ def remove_comment(str):
         output = output + "RVOP(" + op[i] + ", {\n"
         substr = impl[i].split("\n")
         for str in substr:
+            str = re.sub("%", "%%", str)
```

Comment:
The substitution should only apply outside of program comments, where it can retain its meaningful context.